extensions/github-repository-search/src/index.tsx

```diff
@@ -121,7 +179,7 @@ function Actions(props: { repository: Repository; onVisit: (repository: Reposito
             icon={{ source: "project.png", tintColor: Color.PrimaryText }}
             title="Open Projects"
             url={`${props.repository.url}/projects`}
-            shortcut={{ modifiers: ["cmd", "shift"], key: "p" }}
+            shortcut={{ modifiers: ["cmd", "shift", "opt"], key: "p" }}
```

Comment:
this is quite a lot, open to suggestions

Reply:
I guess it's fine.